bindings/matrix-sdk-ffi/src/room.rs

```diff
@@ -810,6 +810,23 @@ impl Room {
 
         Ok(())
     }
+
+    /// Attempt to manually resend messages that failed to send due to issues
+    /// that should now have been fixed. This is useful for example, when
+    /// there's a
+    /// `SessionRecipientCollectionError::VerifiedUserChangedIdentity` error;
+    /// the user may have re-verified on a different device and would now
+    /// like to send the failed message that's waiting on this device.
+    ///
+    /// # Arguments
+    ///
+    /// * `transaction_id` - The send queue transaction identifier of the local
+    ///  echo the send error applies to.
```

Comment:
Bad copy pasto, nowhere else we do mention a "send error".

Reply:
🍝